draft-ietf-quic-http.md

```diff
@@ -246,6 +246,44 @@ DISCUSS:
 do we need to retain compatibility with HTTP/2's HPACK? 
 
 
+### The CONNECT Method
+
+In HTTP/1.x, the pseudo-method CONNECT ({{!RFC7231}}, Section 4.3.6) is used to 
+convert an HTTP connection into a tunnel to a remote host. CONNECT is primarily 
+used with HTTP proxies to establish a TLS session with an origin server for the 
+purposes of interacting with "https" resources. In HTTP/2, the CONNECT method is 
+used to establish a tunnel over a single HTTP/2 stream to a remote host for 
+similar purposes. 
```

Comment:
I would move the second sentence to be the first, this establishes the purpose.  Then you can say how: In HTTP/1.x, CONNECT is used to convert an entire connection into a tunnel to a remote host; in HTTP/2 a single stream is converted into a tunnel.

Then the rest flows naturally.